test/models/viewmodels/decoder/CHIEFErrorSpec.scala

```diff
@@ -38,12 +41,39 @@ class CHIEFErrorSpec extends UnitSpec {
       }
     }
 
-    "contain correct prefix for all message keys" in {
-
-      val expectedPrefix = "error.chief."
-
-      CHIEFError.allErrors.foreach { error =>
-        error.messageKey must include(expectedPrefix)
+    "have translations for all errors" in {
+      val messages = UnitViewSpec.realMessagesApi.preferred(FakeRequest().withCSRFToken)
+
+      val chiefErrorsNames = Seq(
+        "MucrAlreadyShut",
```

Comment:
Nit pick - couldn't we read these values out of the csv file(s)?   That way we ensure the message file is in sync without having to change the test.